docs/api/hardhat/getting-started.md

```diff
@@ -18,48 +18,51 @@ If you are using Vyper, check out the [Vyper plugin documentation](./hardhat-zks
 
 ## Prerequisites
 
-For this tutorial, the following programs must be installed:
+- You have `yarn` package manager installed, npm examples will be added soon.
+- You are already familiar with deploying smart contracts on zkSync. If not, please refer to the first section of the [quickstart tutorial](../../dev/building-on-zksync/hello-world.md).
+- You already have some experience working with Ethereum.
+- You have a wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync as well as deploying smart contracts. We recommend using [our faucet from the zkSync portal](https://goerli.portal.zksync.io/faucet).
+- You know how to get your [private key from your MetaMask wallet](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
 
-- `yarn` package manager. `npm` examples will be added soon.
-- A wallet with sufficient Göerli `ETH` on L1 to pay for bridging funds to zkSync as well as deploying smart contracts. We recommend using [our faucet from the zkSync portal](https://goerli.portal.zksync.io/faucet).
-
-## Project setup
+## Setup
 
 1. To initialize the project and install the dependencies, run the following commands in the terminal:
-::: code-tabs
 
-@tab:active yarn
+::: code-tabs
 
-```bash
+@tab:active folder
```

Comment:
Remove code tabs here. We're only doing yarn so we only need one tab. Don't use tabs for anything else except code, ie `yarn` versus `npm` -> not **folder**, do this separately.